python/ray/train/tests/_lightning_utils.py

```diff
@@ -0,0 +1,79 @@
+import pytorch_lightning as pl
+import torch.nn as nn
+import torch
+from torch.utils.data import DataLoader
+
+
+class LinearModule(pl.LightningModule):
```

Comment:
Can we move these to the test file if these are only used for testing?

Reply:
Actually these model class will be used for the other tests in the following PRs. As the definition of a lightning module is much longer than that of a torch module, I extracted these modules for shared usage to avoid redundant code.